<ui version="4.0" stdsetdef="1" >
  <class>RestartOrExitDialog</class>
  <widget class="QDialog" name="RestartOrExitDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>385</width>
        <height>229</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>New Game or Exit</string>
    </property>
    <layout class="QGridLayout" >
      <item row="2" column="0" >
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QPushButton" name="newGameButton" >
              <property name="text" >
                <string>New Game</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Starts a new game</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="doNothingButton" >
              <property name="text" >
                <string>Do Nothing</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Do nothing. Game buttons will be available</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="exitButton" >
              <property name="text" >
                <string>Exit</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Quit KsirK</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="1" column="0" >
        <spacer name="spacer1" >
          <property name="sizeHint" >
            <size>
              <width>31</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="messageLabel" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text" >
            <string>%1 won! Do you want to play again?</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
</ui>
